modules/examiner/test/TestPlan.md

```diff
@@ -8,4 +8,15 @@
 4. Check "Clear Filters" button works well.
    [Automation Testing]
 5. Test Add a new examiner.
-   [Automation Testing]
+   [Automation Testing]  
+    a. Check that if you try to add an examiner with no name or no site, a warning message appears.  
+    b. A success message should appear when the examiner is created.  
+    c. Check that the new examiner appears on the examiner list. 
+6. Test Edit Examiner.  
+    a. Click on an existing examiner's name and check that it takes you to the "Edit Examiner" page.    
+    b. Fill out the BMI and Radiology Review sections, as well as the dates, and press "Save". There
+    should be no error.  
+    c. Press the "Reset" button and check that the information you filled out in step B appears in the Change Log.    
+    d. Try to change either the BMI or Radiology Review back to the "N/A" option and check that an error message appears.     
+    e. Change the status of either BMI or Radiology Review to "Certified" and return to the examiner list page. Check
+    that the examiner now has that certification listed.   
```

Comment:
I think our convention is to put `[Manual Testing]` after points like both 5 and 6point but not sure  - it should be clear from other test plans.